src/systems/assembly/src/main/java/dk/sdu/se23g6/arch/projecttitle/models/order/dto/OrderStepDTO.java

```diff
@@ -0,0 +1,16 @@
+package dk.sdu.se23g6.arch.projecttitle.models.order.dto;
+
+import dk.sdu.se23g6.arch.projecttitle.models.order.StepStatus;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@RequiredArgsConstructor
+public class OrderStepDTO {
+    private String orderId;
```

Comment:
Why does the OrderStepDTO have a orderId, while the OrderStep has none. Both should have the same class fields.